src/signed-api-fetch/data-fetcher.ts

```diff
@@ -64,14 +63,8 @@ export const runDataFetcher = async () => {
     });
   }
 
-  const urls = uniq(
-    Object.keys(config.chains).flatMap((chainId) =>
-      Object.entries(signedApiUrlStore[chainId]!).flatMap(([airnodeAddress, baseUrl]) => `${baseUrl}/${airnodeAddress}`)
-    )
-  );
-
   return Promise.allSettled(
-    urls.map(async (url) =>
+    signedApiUrlStore.map(async (url) =>
```

Comment:
Mhh, are these URLs unique per provider? The previous implementation looked correct to me.

Reply:
The previous implementation wasn't per provider, I've made these per provider now 👌